src/main/java/org/c4marathon/assignment/global/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package org.c4marathon.assignment.global.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+public class GlobalExceptionHandler {
```

Comment:
```suggestion
public class GlobalExceptionHandler extends ResponseEntityExceptionHandler{
```

`ResponseEntityExceptionHandler`를 사용하는 방안도 고민해보시면 좋을 것 같아요.
많은 보일러 플레이트를 줄일 수 있고 에러 메시지에 관한 RFC 9457에 대응하여 에러를 핸들링 할 수 있습니다.